assets/app/view/game/company.rb

```diff
@@ -8,12 +8,30 @@ class Company < Snabberb::Component
       needs :selected_company, default: nil, store: true
       needs :game, store: true
       needs :tile_selector, default: nil, store: true
-      needs :inline, default: true
+      needs :display, default: 'inline-block'
+      needs :table, default: false
 
       def selected?
         @company == @selected_company
       end
 
+      def purchasable?
+        !@company.owner || @game.round.operating? && @company.owner.player?
+      end
+
+      def ability_usable?
+        return if (@company.all_abilities.map(&:type) & View::Game::Round::Operating::ABILITIES).empty?
+
+        @game.round.can_act?(@company.owner)
```

Comment:
i don't think that's right because players can use abilities at any time

Reply:
ok, I’ll limit it to corps then.